src/ReactiveUI-Blend.nuspec

```diff
@@ -23,11 +23,11 @@
       </group>
       <group targetFramework="uap10.0">
         <dependency id="reactiveui-core" version="$version$" />
-        <dependency id="System.Diagnostics.Debug" version="4.0.10" />
         <dependency id="Microsoft.Xaml.Behaviors.Uwp.Managed" version="1.1.0" />
         <dependency id="Rx-Core" version="2.2.5" />
         <dependency id="Rx-Interfaces" version="2.2.5" />
         <dependency id="Rx-Linq" version="2.2.5" />
+        <dependency id="System.Diagnostics.Debug" version="4.0.10" />
```

Comment:
@kentcb I had modified the existing test that was for this to use an interface as opposed to a var
I made 4 commits on this issue, sorry to confuse things but Appveyor was complaining and I wasn't sure that I had satisfied all the requirements.
I am about to fly to China I will look to see if I can come up with some additional Tests to better prove the operation of the ViewLocator whilst there.
I hope this doesn't hold back the ability to merge this commit.